packages/client/lib/cluster/multi-command.ts

```diff
@@ -224,6 +230,19 @@ export default class RedisClusterMultiCommand<REPLIES = []> {
     return this;
   }
 
+  addScript(
+    firstKey: RedisArgument | undefined,
+    isReadonly: boolean | undefined,
+    script: RedisScript,
+    args: CommandArguments,
+    transformReply?: TransformReply
+  ) {
+    this.#setState(firstKey, isReadonly);
```

Comment:
just inline this #setState/#multi.AddScript in above usage, and remove this function